README.md

```diff
@@ -531,19 +538,66 @@ go get -u github.com/stripe/stripe-go/v79@v77.1.0-beta.1
 ```
 
 > **Note**
-> There can be breaking changes between beta versions. 
+> There can be breaking changes between beta versions.
 
 We highly recommend keeping an eye on when the beta feature you are interested in goes from beta to stable so that you can move from using a beta version of the SDK to the stable version.
 
 If your beta feature requires a `Stripe-Version` header to be sent, set the `stripe.APIVersion` field using the `stripe.AddBetaVersion` function to set it:
 
 > **Note**
-> The `APIVersion` can only be set in beta versions of the library. 
+> The `APIVersion` can only be set in beta versions of the library.
 
 ```go
 stripe.AddBetaVersion("feature_beta", "v3")
 ```
 
+### Raw Request
+
+If you would like to send a request to an API that is:
+
+- not yet supported in stripe-go (like any `/v2/...` endpoints), or
+- undocumented (like a private beta), or
+- public, but you prefer to bypass the method definitions in the library and specify your request details directly
+
+You can use the `RawRequest` method on the `stripe` backend.
+
+```go
+import "github.com/stripe/stripe-go/v79"
```

Comment:
I may also add a simplified example here tomorrow before merge, but this example also works